trunner/testcase.py

```diff
@@ -219,6 +227,7 @@ def __init__(
         super().__init__(name, target, timeout, exec_cmd, use_sysexec, status)
         self.harness = UnitTestHarness.harness
         self.unit_test_results = []
+        self.type = 'unit'
```

Comment:
please note that this could be a class const, not an instance variable (it just describes the class)

Reply:
solved with `is_type` logic removed